ui/src/app/base/sagas/websockets.ts

```diff
@@ -481,14 +481,14 @@ export function* handleMessage(
         // can be 0 when requesting a model with an id of 0.
         if (error || error === 0) {
           yield* put({
-            meta: { item },
+            meta: { item, modelPK: action.meta.modelPK },
```

Comment:
It's possible we don't need `item` here anymore if we're explicitly setting the id.

Reply:
Leaving it in for backwards compatibility